static/app/types/event.tsx

```diff
@@ -493,7 +493,29 @@ interface OtelContext extends Partial<Record<OtelContextKey, unknown>>, BaseCont
   [OtelContextKey.RESOURCE]?: Record<string, unknown>;
 }
 
+export enum ThreadPoolInfoContextKey {
+  MIN_WORKER_THREADS = 'min_worker_threads',
```

Comment:
values added according to https://github.com/getsentry/sentry-dotnet/blob/01ca1d4e2e850d33984929668558068376c4aa2e/src/Sentry/Internal/ThreadPoolInfo.cs#L34-L39